docs/core/diagnostics/logging-and-tracing.md

```diff
@@ -0,0 +1,96 @@
+---
+title: .NET Core Logging and Tracing
+description: An introduction to .NET Core Logging and Tracing.
+author: stmaclea
+ms.author: stmaclea
+ms.date: 8/5/2019
+ms.topic: article
```

Comment:
Please update the metadata here with the same suggestions as the previous article. However, note that this isn't an overview article. You can delete `ms.topic` completely since we have it configured to default to `conceptual`.

```suggestion
```